dnnlib/comet_utils.py

```diff
@@ -0,0 +1,21 @@
+try:
+    from comet_ml import Experiment
+    comet_installed = True
+except:
+    print("Comet not installed")
+    comet_installed = False
+import tensorflow as tf
+import json
+
+
+class CometLogger():
+    def __init__(self):
+        self.comet_installed = comet_installed
```

Comment:
Why is comet_installed both a module variable and an instance variable?

Reply:
Is it better to use the module variable initialized in CometLogger.py inside training_loop.py?

Reply:
Module variable is set because we cannot import comet inside a class (Otherwise TensorFlow will be imported first and we will have issues with comet). I changed the logic now. Now I have one global variable and a  '_logging'  instance variable which is True only when comet is installed and API is configured. !